src/logging.cc

```diff
@@ -144,6 +144,8 @@ GLOG_DEFINE_string(alsologtoemail, "",
                    "in addition to logfiles");
 GLOG_DEFINE_bool(log_prefix, true,
                  "Prepend the log prefix to the start of each log line");
+GLOG_DEFINE_bool(log_year_in_prefix, true,
+                 "The log prefix should be contain the year in each line of output");
```

Comment:
The same grammar problems...
Suggested phrasing:
```
Include the year in the log prefix
```

Reply:
I modified the text according to your suggestion.

Reply:
My 2 suggestions (for `logging.h.in` and for `logging.cc`) were not identical.
They were somehow mirroring style of descriptions of other options, e.g. `log_prefix`.

The comments in `logging.h.in` contain `Set whether ` fragment, e.g.:
```
// Set whether the log prefix should be prepended to each line of output.
DECLARE_bool(log_prefix);
```
But descriptions in `logging.cc` do not contain `Set whether ` fragment, e.g.:
```
GLOG_DEFINE_bool(log_prefix, true,
                 "Prepend the log prefix to the start of each log line");
```
So you applied my suggestion for `logging.h.in`, but not my suggestion for `logging.cc`.